zbsprimarystorage/zbsprimarystorage/zbsutils.py

```
@@ -232,7 +233,7 @@ def cbd_to_nbd(desc, port, install_path):
     cmd = "qemu-nbd -D %s -f raw -p %d --fork %s_%s_:%s" % (
         desc, port, install_path, ZBS_USER_NAME, ZBS_CLIENT_CONF_PATH)
     logger.debug(cmd)
-    os.system(cmd)
+    subprocess.run(cmd, shell=True, check=False)
```

Ruff (0.15.5)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)